README.md

```diff
@@ -22,10 +22,10 @@ None
 
 |Field Name        |Description                                                                                                                                                                                                                                                    |
 |:-----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-|Source            |The source to display in the legend for the CPU data.                                                                                                                                                                                                          |
-|Process Name Regex|A regular expression to match the name of the process.                                                                                                                                                                                                         |
-|Process Path Regex|A regular expression to match the full path of the process.                                                                                                                                                                                                    |
-|Process CWD Regex |A regular expression to match the current working directory of the process.                                                                                                                                                                                    |
+|Source            |The source to display in the legend for the CPU data.|
+|Process Parameter Option|Parameter option for process matching. Parameter can be any of name/path/cwd/args regex. Choose the parameter option as Process Name Regex or Process Path Regex or Process CWD Regex or Process Args Regex |
```

Comment:
Change "Choose the parameter option as Process Name Regex or Process Path Regex or Process CWD Regex or Process Args Regex" to
"Choose the parameter option as Process Name Regex, Process Path Regex, Process CWD Regex or Process Args Regex"